mod-audit/src/test/java/org/folio/AuditSimulation.scala

```diff
@@ -0,0 +1,43 @@
+package org.folio
+
+import com.intuit.karate.gatling.PreDef._
+import io.gatling.core.Predef._
+import org.apache.commons.lang3.RandomUtils
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class AuditSimulation extends Simulation {
+
+  def generateTenantId(): String = {
+    val constantString = "testtenant"
+    val randomLong = RandomUtils.nextLong
+    constantString + randomLong
+  }
+
+  val protocol = karateProtocol(
+    "/_/proxy/tenants/{tenant}" -> Nil,
+    "/_/proxy/tenants/{tenant}/modules" -> Nil,
+    "/_/proxy/tenants/{tenant}/install" -> Nil,
+    "/circulation/loans/{loanId}" -> Nil,
+    "/audit-data/circulation/logs" -> Nil,
+    "/circulation/check-out-by-barcode" -> Nil,
+    "/circulation/check-in-by-barcode" -> Nil,
+  )
+  protocol.runner.systemProperty("testTenant", generateTenantId())
+
+  val before = scenario("before")
+    .exec(karateFeature("classpath:firebird/mod-audit/mod-audit-junit.feature"))
+  val create = scenario("create")
+    .repeat(10) {
+      exec(karateFeature("classpath:thunderjet/mod-orders/features/checkInCheckOutEvent.feature"))
```

Comment:
guessing this path will be corrected to firebird and mod-audit

Reply:
done, I forgot to push the changes